openpype/hosts/maya/plugins/publish/validate_resolution.py

```diff
@@ -0,0 +1,81 @@
+import pyblish.api
+from openpype.pipeline import (
+    PublishValidationError,
+    OptionalPyblishPluginMixin
+)
+from maya import cmds
+from openpype.pipeline.publish import RepairAction
+from openpype.hosts.maya.api import lib
+from openpype.hosts.maya.api.lib import reset_scene_resolution
+
+
+class ValidateSceneResolution(pyblish.api.InstancePlugin,
```

Comment:
Can we add project settings for this plug-in so studios can tweak the enabled, optional, active state from settings?